src/qgis_venv_creator/create_qgis_venv.py

```diff
@@ -0,0 +1,446 @@
+            "QGIS_INSTALLATION_SEARCH_PATH_PATTERN"
+        )
+        qgis_installations = list(cls._find_qgis_installations(custom_search_path_pattern))
+
+        print("Found following QGIS installations from the system. Which one to use for development?")
+        for i, path in enumerate(qgis_installations):
+            print(f"  {i+1} - {path}")
+        print(f"  {i+2} - Custom")
+        choose_prompt = f"Choose from [{'/'.join(str(i+1) for i in range(i+2))}]"
+        while True:
+            try:
+                selection = int(input(f"  {choose_prompt}: "))
+            except ValueError:
+                print("Invalid selection.")
+                continue
+
+            if selection == len(qgis_installations) + 1:
+                while True:
+                    custom_qgis_path = Path(input("  Give path to QGIS installation: "))
+                    if not cls._is_valid_qgis_path(custom_qgis_path):
+                        print("Invalid qgis installation path")
+                        continue
+                    return custom_qgis_path
+            else:
+                try:
+                    return qgis_installations[selection - 1]
+                except IndexError:
+                    print("Invalid selection")
+                    continue
+
+    @staticmethod
+    def cli_arguments() -> list[CliArg]:
+        return [
+            CliArg(
+                "--qgis-installation",
+                help=(
+                    "Path to the QGIS installation to use for development. "
+                    "Installations made with official msi and Osgeo4W instellers are supported. "
+                    "Give the path to the 'qgis' directory inside the 'apps' directory. "
+                    "If not given, the user is prompted to select one."
+                ),
+                type=Path,
+            ),
+            CliArg(
+                "--qgis-installation-search-path-pattern",
+                help=(
+                    "Custom glob pattern for QGIS installations to be selected. "
+                    "Can be set also with QGIS_INSTALLATION_SEARCH_PATH_PATTERN environment variable."
+                ),
+                type=str,
+            ),
+            CliArg(
+                "--python-executable",
+                help=(
+                    "Path to the Python executable used by the QGIS installation. "
+                    "If not given, the Python executable is searched from the QGIS installation."
+                ),
+                type=Path,
+            ),
+        ]
+
+
+class Windows(MultiQgisPlatform):
+    @classmethod
+    def _find_qgis_installations(cls, custom_search_path_pattern: str | None = None) -> list[Path]:
+        """Find all QGIS installations from the Windows system."""
+
+        possible_qgis_installation_generators = [
+            Path("C:/Program Files").glob("QGIS*/apps/qgis*/"),
+            Path("C:/OSGeo4W/apps").glob("qgis*/"),
```

Comment:
Possible installation path when using OSGeo installer is missing (OSGeo4W64).
```suggestion
            Path("C:/OSGeo4W/apps").glob("qgis*/"),
            Path("C:/OSGeo4W64/apps").glob("qgis*/"),
```

Reply:
Thanks, Added that now. That's so old I even forgot that possibility.